UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Services/FssEventProcessor.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Azure.Messaging;
+using FluentValidation.Results;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using UKHO.ExternalNotificationService.API.Extensions;
+using UKHO.ExternalNotificationService.Common.BaseClass;
+using UKHO.ExternalNotificationService.Common.Configuration;
+using UKHO.ExternalNotificationService.Common.Helpers;
+using UKHO.ExternalNotificationService.Common.Logging;
+using UKHO.ExternalNotificationService.Common.Models.EventModel;
+using UKHO.ExternalNotificationService.Common.Models.Request;
+using UKHO.ExternalNotificationService.Common.Models.Response;
+
+namespace UKHO.ExternalNotificationService.API.Services
+{
+    public class FssEventProcessor : EventProcessorBase , IEventProcessor
+    {
+        private readonly IFssEventValidationAndMappingService _fssEventValidationAndMappingService;
+        private readonly ILogger<FssEventProcessor> _logger;
+        private List<Error> _errors;
+        private readonly IAzureEventGridDomainService _azureEventGridDomainService;
+
+        public string EventType => EventProcessorTypes.FSS;
+
+        public FssEventProcessor(IFssEventValidationAndMappingService fssEventValidationAndMappingService,
+                                 ILogger<FssEventProcessor> logger,
+                                 IAzureEventGridDomainService azureEventGridDomainService)
+                                : base(azureEventGridDomainService)
+        {
+            _fssEventValidationAndMappingService = fssEventValidationAndMappingService;
+            _logger = logger;
+            _azureEventGridDomainService = azureEventGridDomainService;
```

Comment:
Remove this unused assignment.

Reply:
Thanks for highlighting, Removed this unused assignment.